x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/actions_menu.test.tsx

```diff
@@ -59,6 +62,15 @@ describe('When using the Endpoint Details Actions Menu', () => {
     waitForAction = mockedContext.middlewareSpy.waitForAction;
     httpMocks = endpointPageHttpMock(mockedContext.coreStart.http);
 
+    (useUserPrivileges as jest.Mock).mockReturnValue({
```

Comment:
Because the entire module `.../components/user_privileges` is being mocked, do you need to reset `useUserPrivileges()` after each test (in `afterEach()`)?

Reply:
ah, good catch. will add reset.